internal/pubsub/receive_messages.go

```diff
@@ -3,14 +3,27 @@ package pubsub
 import (
 	"cloud.google.com/go/pubsub"
 	"context"
+	"encoding/json"
 	"fmt"
+	"github.com/broadinstitute/revere/internal/cloudmonitoring"
 	"github.com/broadinstitute/revere/internal/configuration"
+	"github.com/broadinstitute/revere/internal/shared"
 )
 
 // receiveOnce should handle and acknowledge a single message; will run
 // asynchronously
 func receiveOnce(config *configuration.Config, msg *pubsub.Message) {
```

Comment:
This function is basically a big print statement right now, it is _not_ how this will end up working.